### ISSUE TITLE
Move timestamp next to username and make it visible

### ISSUE THREAD

**Issue by anitagraham** [gitterHQ/gitter#743]
The UI changes have made the timestamp on each message difficult to find (far to the right) and difficult to see (so pale...).

![gitter conversation](https://cloud.githubusercontent.com/assets/397118/8323062/e6240104-1a71-11e5-9d8f-9e6e2891266c.png)

I suggest that the timestamp be placed immediately to the right of the username. Particularly with short messages on wide screens it's hard to locate the timestamp.
A timestamp besides the name indicates that *name* at *time* said *message*.

I suggest that the timestamp be in a darker colour. If it's worthwhile writing a timestamp it must be worthwhile for the user to see it.

**Comment by MadLittleMods** [comment]
Here is one of the reasons for the removal of the relative "time ago" timestamps:

> It chews up CPU, every minute we're updating the multiple elements in the DOM
>
> [*Source: Mike Bartlett*](https://gitter.im/gitterHQ/gitter?at=55807f63f3db3f6f74b0b59b)

This was also mentioned/suggested [here](https://gitter.im/gitterHQ/gitter?at=5582aed005d28d2a1592fd14) and [here](https://gitter.im/gitterHQ/gitter?at=55807ee01c3ba5ef5bfed5d7).

**Comment by anitagraham** [comment]
Yes, I understand this, which is why I didn't include it in this issue.
Thank you.

**Comment by MadLittleMods** [comment]
Color contrast separated it into its own issue here at #771 with specifics on visual contrast.
